lib/cli/ares.js

```diff
@@ -57,7 +57,7 @@ async function runCmd(bin, args = []) {
   const _bin = path.resolve(LG_WEBOS_TV_SDK_HOME, ...WEBOS_CLI_PATH_PREFIX, bin);
   log.info(`Running command: ${_bin} ${args.join(' ')}`);
   try {
-    return await exec(_bin, args);
+    return await exec(_bin, args, {shell: true});
```

Comment:
nits: Gives true only when `system.isWindows()`?

Reply:
yeah. i'm kinda thinking we should probably set this in teen process as a default true for the windows case, but want to see if it fixes the issue first.